extensions/mutiny/runtime/src/main/java/io/quarkus/mutiny/runtime/MutinyInfrastructure.java

```diff
@@ -36,7 +36,7 @@ public boolean getAsBoolean() {
                  * calling from a Vert.x event-loop context / thread.
                  */
                 String threadName = Thread.currentThread().getName();
-                return !threadName.contains("vertx-eventloop-thread-");
+                return !threadName.startsWith("vert.x-eventloop-thread-");
```

Comment:
Couldn't we have a constant in Vert.x or an API to test that?

Reply:
No, because this would add dependency from this extension to Vert.x core. I had initially tried writing the test with Vert.x APIs, but that didn't work here.

Reply:
Note that this naming scheme has been consistent over the years and is not changing anytime soon.

Reply:
We want to avoid having an explicit dependency on Vert.x. I believe there is a comment about that. 
That being said, we can create a constant.

Reply:
Added a constant